projects/js-packages/charts/package.json

```diff
@@ -39,17 +39,27 @@
 			"jetpack:src": "./src/index.ts",
 			"import": "./dist/mjs/index.js",
 			"require": "./dist/cjs/index.js",
-			"types": "./dist/index.d.ts"
-		},
-		"./components/*": {
-			"import": "./dist/mjs/components/*/index.js",
-			"require": "./dist/cjs/components/*/index.js"
+			"types": "./dist/types/index.d.ts"
 		},
 		"./visx/*": {
+			"jetpack:src": "./src/visx/*",
 			"import": "./dist/mjs/visx/*/index.js",
-			"require": "./dist/cjs/visx/*/index.js"
+			"require": "./dist/cjs/visx/*/index.js",
+			"types": "./dist/types/visx/*/index.d.ts"
 		},
-		"./style.css": "./dist/mjs/style.css"
+		"./providers/*": {
+			"jetpack:src": "./src/providers/*",
+			"import": "./dist/mjs/providers/*/index.js",
+			"require": "./dist/cjs/providers/*/index.js",
+			"types": "./dist/types/providers/*/index.d.ts"
+		},
+		"./style.css": "./dist/mjs/style.css",
+		"./*": {
```

Comment:
Make the import shorter, so it'd be `@automattic/charts/line-chart` instead of `@automattic/charts/components/line-chart`.